code/game/machinery/doors/airlock.dm

```diff
@@ -1029,6 +1029,29 @@ About the new airlock wires panel:
 
 	return
 
+/obj/machinery/door/airlock/attack_alien(mob/living/carbon/alien/humanoid/user)
+	if(isElectrified())
+		shock(user, 100)
```

Comment:
Damage is based on how much power is in the net.
shock() calls electrocute_mob() which deals damage based on the power in the net attached to the APC (pretty sure, haven't bothered checking the code)